### ISSUE TITLE
Why is the callback executed twice?

### ISSUE THREAD

**Issue by sylvoslee** [StratoDem/sd-material-ui#79]
when i click checkbox，callback executed twice，print  is double，is this correct?

`@app.callback(
    dash.dependencies.Output('output8', 'children'),
    [dash.dependencies.Input('input8', 'checked')])
def use_clickbox(checkbox):
    print('First execution')
    if checkbox:
        print('Second execution')
        return ['Box is checked']
    else:
        print('Second execution')
        return ['Box is unchecked']`

**Comment by mjclawar** [comment]
Sorry, going to need a little more information on this:

```python
@app.callback(
    dash.dependencies.Output('output8', 'children'), 
    [dash.dependencies.Input('input8', 'checked')]) 
def use_clickbox(checkbox): 
    print('First execution') 
    
    if checkbox: 
        print('Second execution') 
        return ['Box is checked'] 
    else: 
        print('Second execution') 
        return ['Box is unchecked']
```

> print is double

What do you mean double? Can you just copy-paste the logs into this issue?

**Comment by sylvoslee** [comment]
I mean when the callback is triggered， the print('First execution') was executed twice.
127.0.0.1 - - [05/Sep/2018 13:36:58] "POST /_dash-update-component HTTP/1.1" 200 -
127.0.0.1 - - [05/Sep/2018 13:36:58] "POST /_dash-update-component HTTP/1.1" 200 -
First execution
Second execution
127.0.0.1 - - [05/Sep/2018 13:37:20] "POST /_dash-update-component HTTP/1.1" 200 -
First execution
Second execution
127.0.0.1 - - [05/Sep/2018 13:37:20] "POST /_dash-update-component HTTP/1.1" 200 -

**Comment by mjclawar** [comment]
Patched in #80 and released in [`sd-material-ui==2.16.1`](https://github.com/StratoDem/sd-material-ui/releases/tag/v2.16.1)

**Comment by sylvoslee** [comment]
It still exists in SDToggle,SDRadioButtonGroup，can you fix it，thank you
